adding_species_manuscript.tex

```diff
@@ -846,18 +849,20 @@ \section*{Conclusion}
 We also discuss special considerations for collecting relevant information from the literature,
 and what to do if some of that information is not available.
 Because this process is quite error-prone,
-we encourage everyone implementing simulations to have their
-parameter choices and implementation reviewed by at least one other
-researcher.
-These steps can be followed if you are implementing a simulation independently for your own use,
-or, as we encourage, when you add your simulation code to \stdpopsim,
-to ensure its robustness and make it available for future standardized research.
+we encourage wider adoption of ``code review'',
+i.e., for researchers implementing simulations to have their
+parameter choices and implementation reviewed by at least one other researcher.
+%% PLR: It's not totally clear which "these steps" are, and they are actually required for stdpopsim?
```

Comment:
"These steps" refers to our guidelines, and not just the QC part. This statement is important, but could be clarified by changing "These steps" to "The guidelines we provide in this paper". Some of these guidelines are strict requirements of stdpopsim and some are not. I think it's okay not to make this distinction here.

Reply:
Ah, I see. I was shying a little away from saying "everyone please add all your simulation code to stdpopsim" since it can be hard to find QC'ers. But, how's this?

Reply:
Looks good. I do think we should be encouraging people to add their code to stdpopsim. If they read this paper, then they know what our requirements and standards are, so I don't think we are risking open the door too wide here.